### PR TITLE
Generated files for che memberstatus condition

### PR DESCRIPTION
**Pull request by rajivnathan**
Related PRs:
https://github.com/codeready-toolchain/member-operator/pull/231
https://github.com/codeready-toolchain/api/pull/210

### CODE REVIEW

File: deploy/registration-service/registration-service.yaml [hunk 1]
```diff
@@ -167,10 +167,6 @@ objects:
                     configMapKeyRef:
                       name: registration-service
                       key: verification.message_template
-              resources:
-                requests:
-                  cpu: "50m"
-                  memory: "100M"
```
Comment (alexeykazakov) [inline deploy/registration-service/registration-service.yaml:173]: Why it's being deleted?

Comment (rajivnathan) [reply deploy/registration-service/registration-service.yaml:173]: I'm not sure, I don't see anything in my changes that would cause that.

Comment (alexeykazakov) [reply deploy/registration-service/registration-service.yaml:173]: Then please bring it back :)

Comment (MatousJobanek) [reply deploy/registration-service/registration-service.yaml:173]: The reason is that the resource requests are not defined in the registration-service repo.

Comment (MatousJobanek) [reply deploy/registration-service/registration-service.yaml:173]: fixed in https://github.com/codeready-toolchain/registration-service/pull/163

**Review by MatousJobanek — APPROVED** [review]

File: deploy/registration-service/registration-service.yaml (hunk 1 above)
Comment (alexeykazakov) [reply deploy/registration-service/registration-service.yaml:173]: Good catch. BTW I think we don't really need that much resources (there are three replicas so the total would be 150m/300M) but let's leave it for another PR :)

Comment (rajivnathan) [reply deploy/registration-service/registration-service.yaml]: Thanks @MatousJobanek. @alexeykazakov it's back now after pulling in Matous' change and regenerating.

**Review by alexeykazakov — APPROVED** [review]

**Pull request merged**